ledger/appcow.go

```diff
@@ -240,6 +204,7 @@ func errAlreadyStorage(addr basics.Address, aidx basics.AppIndex, global bool) e
 	return fmt.Errorf("%v has already opted in to app %d", addr, aidx)
 }
 
+// Allocate creates kv storage for a given {addr, aidx, global} and happens on app creation (global) or opting in (local)
```

Comment:
Did you mean "as happens" instead of "and happens"?

Reply:
done